projects/1.20/assets/the-dawn-era/dawnera/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+{
+  "accessibility.dawnera.private": "private",
+  "accessibility.dawnera.public": "public",
+  "action.dawnera.inventory": "inventory",
+  "advancements.dawnera.dont_move.description": "使用望远镜来观察霸王龙",
```

```suggestion
  "advancements.dawnera.dont_move.description": "使用望远镜来观察暴龙",
```
暴龙是学名，建议最好改掉
此类问题（包括其他恐龙的学名）请全文通查并修改